tests/plugins/test_fetch_maven_artifacts.py

```diff
@@ -65,6 +63,7 @@
     'artifact_id': 'jaxb-core',
     'build_id': 472397,
     'checksum': '697317209103338c7c841e327bb6e7b0',
+    'checksum_type': 0,
```

Comment:
Should we use `koji.CHECKSUM_TYPES['md5']` here? (and add `CHECKSUM_TYPES` to our `koji/__init__.py` stub)